src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -387,4 +490,10 @@ public ListBoxModel doFillRepositoryItems(@AncestorInPath SCMSourceOwner context
 
     }
 
+    @Extension public static class GitHubSCMSourceAddition implements GitHubSCMNavigator.GitHubSCMSourceAddition {
```

Comment:
I think this is not needed (and it should be removed from `GitHubSCMNavigator` also).